src/index.js

```diff
@@ -3,6 +3,12 @@ import ReactDOM from 'react-dom/client';
 import './index.scss';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import CatalogPage from './pages/CatalogPage'
```

Comment:
`./pages/CatalogPage` n’existe pas, quand tu démarre l’appli ça ne va pas marcher. Il faut l’effacer ou commenter
```suggestion
// import CatalogPage from './pages/CatalogPage'
```